mobile/lib/features/trade/trade_bottom_sheet_confirmation.dart

```diff
@@ -180,6 +167,52 @@ class TradeBottomSheetConfirmation extends StatelessWidget {
         ? Amount((referralStatus.referralFeeBonus * orderMatchingFee.sats).floor())
         : Amount.zero();
 
+    final description = switch (tradeAction) {
+      TradeAction.closePosition => RichText(
+          text: TextSpan(
+              text:
+                  '\nBy confirming, a closing market order will be created. Once the order is matched, your position will be closed.',
```

Comment:
_closing_ is redundant as it's mentioned in the second half of the sentence. 

```suggestion
                  '\nBy confirming, a market order will be created. Once the order is matched, your position will be closed.',
```